data/ny/legislature/James-Gaughran-089d12fb-2567-487c-8661-d5a930cb0c16.yml

```diff
@@ -1,30 +1,33 @@
 id: ocd-person/089d12fb-2567-487c-8661-d5a930cb0c16
-name: James Gaughran
+name: Steven D. Rhoads
 given_name: James
 family_name: Gaughran
```

Comment:
@bfossen-ce this is a weird one. It looks like this file should belong to james gaughran who left office on 01-01-2023. 

Seperately, Steven Rhoads needs his own file created https://ballotpedia.org/Steven_Rhoads